### PR TITLE
Added Export SDK, a PDF generator API with generous free tier.

### PR DESCRIPTION
**Pull request by rionmartin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,6 +205,7 @@ The definition of developer-first for this repo is:
 *Generating reports, mainly PDFs*
 * [PDFBlade](https://pdfblade.com/#pricing) - HTML to PDF API usage based pricing.
 * [Carbone](https://carbone.io/) - JSON into PDF, DOCX, XLSX, PPTX, ODS... API.
+* [Export SDK](https://exportsdk.com) - PDF generator API with visual template editor and generous free tier.
```
Comment (agamm) [inline README.md:208]:
```suggestion
* [Export SDK](https://exportsdk.com) - PDF generator API with visual template editor.
```

**Review by agamm — CHANGES_REQUESTED** [review]
1st, this isn't an advertizing space :) so it needs to be as objective as possible.
2nd, Please explain why you are adding the product (see [this](https://github.com/agamm/awesome-developer-first/blob/main/CONTRIBUTING.md)) specifically:
> If you are not adding the only player in the category, please post the other competitors and explain why this product is the most awesome from the bunch.

**Comment by agamm** [comment]
@rionmartin thanks for the fix, but did you see the second point in my last comment?

**Comment by rionmartin** [comment]
> @rionmartin thanks for the fix, but did you see the second point in my last comment?

@agamm Hey! I typed out a rather lengthy response to your second point in the comments of the commit. Does this not cover all the bases? If so, what's missing? 

"Export SDK was built by a team of devs who have had to integrate PDF generation into their apps over and over again. Export SDK  makes it quick and easy for developers to generate PDFs for their apps or projects without having to build a custom solution, code templates, and maintain additional services. 

Alternatives include API Template and PDF Blade. Compared to PDF Blade, Export SDK provides a drag and drop template editor so that devs can more quickly design and test templates or pass the task to non-devs to create and maintain the templates.

Compared to API Template, Export SDK is solely focused on generating PDFs rather than being a multipurpose solution. It also provides an objectively easier to use template designer based on flexbox. Where API Template only provides 50 pages per month on its free tier, Export SDK provides 250 pages per month. 

Finally, Export SDK is the only tool built with collaboration in mind, allowing users to create organizations for specific companies or projects and add an unlimited number of users."

**Comment by agamm** [comment]
@rionmartin Got it. I didn't see the PR comments (it's quite hidden). Thanks for the update.

**Review by agamm — APPROVED** [review]

**Pull request merged**